shopfloor/services/checkout.py

```diff
@@ -600,7 +600,7 @@ def _select_lines_from_delivery_packaging(
             )
         if confirm_pack_all:
             # Select all lines and pack them all w/o passing for select_package state
-            self._select_lines(selection_lines)
+            self._select_lines(selection_lines, prefill_qty=packaging.qty)
```

Comment:
This can't be right. A delivery package is scanned to put lines in a new package.

I have the impression this feature is not properly made @simahawk 

* If no lines are selected (i.e. none have a done qty) and no_qty_prefill, then ask the user to select lines => missing
* If no lines are selected (i.e. none have a done qty) and not no_qty_prefill, then confirm_put_all_goods_in_delivery_package => how selection_lines is build? Then on confirmation, select_lines and put in pack.
* If lines are selected (i.e. have a done qty), then don't ask confirmation, don't change quantities, just _create_and_assign_new_packaging

Reply:
This should be fixed in another PR (on bscos)